packit/api.py

```diff
@@ -664,6 +666,23 @@ def sync_status_string(
         else:
             return f"'{source_git}' is up to date with '{dist_git}'."
 
+    def get_dg_repo_path_for_actions(self) -> str:
+        """Get the dg repo path, which should be accessible by an action.
+        If we are running actions inside sandcastle, copy repo in shared dir.
+        """
+        if self.config.command_handler == RunCommandType.sandcastle:
+            dg_repo_dir_for_actions = os.path.join(
+                self.config.command_handler_work_dir, SANDCASTLE_DG_REPO_DIR
+            )
+            copy_tree(
+                str(self.dg.local_project.working_dir),
+                str(dg_repo_dir_for_actions),
+                preserve_symlinks=True,
+            )
```

Comment:
You are copying the dist-git repo to a subdirectory before actions are run, but I don't see a code that would copy it back.

Reply:
I moved distgit workdir below sandcastle shared dir. Copying and syncing back should no more be needed.